### PR TITLE
Set up proper paging

### LINKED ISSUE #75: Pass the total amount of pages for the paged products listing
Currently there's no way to determine the total amount of pages

### PR DESCRIPTION
**Pull request by illright**
Closes #75

### CODE REVIEW

File: innopoints/views/project.py [hunk 1]
```diff
@@ -99,6 +91,61 @@ def list_projects():
...
+
+    db_query = db_query.order_by(Project.creation_time.desc())
+    db_query = db_query.offset(limit * (page - 1)).limit(limit)
+
+    conditional_exclude = ['review_status', 'moderators']
+    if current_user.is_authenticated:
+        conditional_exclude.remove('moderators')
+        if not current_user.is_admin:
+            conditional_exclude.remove('review_status')
+    exclude = ['admin_feedback', 'review_status', 'files', 'image_id',
```
Comment (aabounegm) [inline innopoints/views/project.py]: `review_status` is in both `exclude` and `conditional_exclude`

**Review by aabounegm — COMMENTED** [review]
You still insist on having many endpoints, even though their functionality, request params, and response are almost exactly the same except for one or two params. I don't really like this, but it's up to you

File: innopoints/views/project.py (hunk 1 above)
Comment (aabounegm) [reply innopoints/views/project.py]: also, why exclude the image?

Comment (illright) [reply innopoints/views/project.py]: The `image_id` is excluded in favor of the `image_url` property

**Comment by illright** [comment]
As for separating the ongoing and past projects listing into two endpoints, I did it because one is paged and the other is not, and cramming all this functionality seemed a little overly complicated.

**Review by aabounegm — APPROVED** [review]
So we are assuming that ongoing projects are not so many that they deserve to be paginated?

**Comment by illright** [comment]
Yeah

**Pull request merged**